livekit/src/rtc_engine/rtc_session.rs

```diff
@@ -604,6 +604,7 @@ impl SessionInner {
                         });
                     }
                     proto::data_packet::Value::Speaker(_) => {}
+                    proto::data_packet::Value::Transcription(_) => {}
```

Comment:
Waiting on client team to solidify the api before implementing receiving a transcription